### ISSUE TITLE
[Feature Request] Thumbnail Downloads Pause / Cancel

### ISSUE THREAD

**Issue by itoffie** [CarlosEsco/Neko#440]
### Why/User Benefit/User Problem
pause or cancel on going downloads on a certain series instead of re arranging or manually canceling them on the chapter page
### What/Requirements
instead of a list it will show you the mamga thumbnail and the ongoing downloads only are listed inside.

on the download page where thumbnails are located holding the thumbnail will select it and give you the option to pause or to cancel ongoing downloads.

you know that moment you queued the first manga with 100+ ch to download and you found other mangas with just 5 ch or 10 and you want to download it first but you had to scroll all the way to the bottom just to re order or go to the current 100+ ch manga page just to cancel all.

reorder does not do the job well, it downloads newest to oldest updates ( my 100+ ch download has been inserted in the middle of download list after reorder and i had to cancel all ongoing downloads on that manga manually because theres no pause or cancel all downloads on that certain manga only ) instead of queued first vice versa

**Comment by CarlosEsco** [comment]
So you want an option to pause or cancel individual downloads?

**Comment by itoffie** [comment]
yes like i need to pause this manga to download other manga instead of re ordering or cancelling ongoing download just to download other in queue
